### ISSUE TITLE
Failed to activate the epitech-header-2018 package

### ISSUE THREAD

**Issue by cyrilcolinet** [mrlizzard/Epitech-HEADER-2018#1]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.24.0 x64
**Electron**: 1.8.2
**OS**: "Arch Linux"
**Thrown From**: [epitech-header-2018](https://github.com/mrlizzard/Epitech-HEADER-2018) package 1.0.0

### Stack Trace

Failed to activate the epitech-header-2018 package

```
At unexpected identifier in /home/cyrilcolinet/Tools/epitech-header-2018/lib/epitech-header.coffee:62:43

SyntaxError: unexpected identifier
at /home/cyrilcolinet/Tools/epitech-header-2018/lib/epitech-header.coffee:62:43
```

### Commands

```
```

### Non-Core Packages

```
atom-terminal 0.8.0 
autocomplete-clang 0.11.4 
electron-highlighter-syntax 1.15.0 
epitech-header-2018 1.0.0 
file-icons 2.1.17 
formatter-coffeescript 0.1.1 
linter-coffeescript 1.0.0 
markdown-toc 0.4.2 
minimap 4.29.8 
open-terminal-here 2.3.1 
```

**Comment by cyrilcolinet** [comment]
Fixed

**Comment by cyrilcolinet** [comment]
Fixed since [7966c70](https://github.com/mrlizzard/Epitech-HEADER-2018/commit/7966c701f654ec71dee06bbb4cb5e7766481d3eb) commit.
